### PR TITLE
Update README for containers

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

File: container/README.md [hunk 1]
```diff
@@ -6,29 +6,29 @@
...
 A Redis container is required to run and schedule faf actions from the web UI (see below).
 Communication between ABRT Analytics, the database and Redis is ensured by running all three containers
 in the same podman pod.
 
-`podman volume create <volume_name>`  
-`podman pod create -p 5432:5432 -p 6379:6379 -p 8080:8080 --name <pod_name>`  
-`podman run --pod <pod_name> -v <volume_name>:/var/lib/pgsql/data -e POSTGRESQL_ADMIN_PASSWORD=scrt --name db -dit abrt/postgres-semver`
+`podman volume create faf-volume`
+`podman pod create -p 5432:5432 -p 6379:6379 -p 8080:8080 --name faf-pod`
+`podman run --pod faf-pod -v faf-volume:/var/lib/pgsql/data -e POSTGRESQL_ADMIN_PASSWORD=scrt --name db -dit quay.io/abrt/faf-db`
```
Comment (mgrabovsky) [inline container/README.md:17]: Can you please convert all these blocks to the indented form?
```suggestion
    podman volume create faf-volume
    podman pod create -p 5432:5432 -p 6379:6379 -p 8080:8080 --name faf-pod
    podman run --pod faf-pod -v faf-volume:/var/lib/pgsql/data -e POSTGRESQL_ADMIN_PASSWORD=scrt --name db -dit quay.io/abrt/faf-db
```

**Review by mgrabovsky — CHANGES_REQUESTED** [review]
Looks OK, just one small suggestion while we're at it.

By the way, since when do we publish on quay.io?

File: container/README.md (hunk 1 above)
Comment (msrb) [reply container/README.md]: Good point -- updated.

**Comment by msrb** [comment]
> By the way, since when do we publish on quay.io?

That is a good question and I am trying to figure it out :) The namespace already exists, and the images were probably pushed to both docker.io and quay.io by some automation... 9 months ago. But not since then.

**Review by mgrabovsky — APPROVED** [review]

**Pull request merged**